### DISCUSSION TITLE
X86 (32 bit) Linux compatibility

### DISCUSSION THREAD

**Discussion by hellc** [netbirdio/netbird#372] [Issue Triage]
Is it possible to build and release a 32bit Linux binaries for netbird cli? I would like to run it on alpine x86.

**Comment by mlsmaycon** [comment]
Hi @hellc I would have to test if all the functions are working properly but it should be possible to build it for 32bits.

**Reply by hellc** [reply]
Great, let us know (if possible) here. 
I would also like to contribute as an external tester then. 
Thank you for your work!

**Reply by mlsmaycon** [reply]
the PR #374 will add the 32 bits build. 

I ran some tests on ubuntu 32 bits and it seem to be all good.

**Reply by mlsmaycon** [reply]
should be on our next release

**Comment by mlsmaycon** [comment]
@hellc we have added the 32 binary. You can test it using our binary installation path: https://netbird.io/docs/getting-started/installation#binary-install
